apps/website/src/pages/docs/tooltip.mdx

```diff
@@ -9,18 +9,43 @@ group: core
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tooltip' />
-
 <LiveExample src='Tooltip.main.tsx' truncate={false}>
   <AllExamples.TooltipMainExample client:load />
 </LiveExample>
 
-A tooltip is a small container that typically appears above the element you are hovering with your cursor. The tooltip displays a short, helpful text message. Tooltips should never be used to display critical information, or as a means to navigate an interface. Because mobile devices do not have a hover state, use caution when using tooltips in desktop apps that have a mobile counterpart. There is no determined way of replacing tooltips in mobile devices. In some cases, a single short tap can display a tooltip on a touch display.
+A tooltip is a small container that typically appears above the element you are hovering with your cursor with a short, helpful text message. You can use up to 256 characters. Content other than text such as pictures, tables, etc, are not recommended.
+
+Mobile devices does not have a hover state, caution when using tooltips in desktop apps that have a mobile counterpart. There is no determined way of replacing tooltips in mobile devices.
+
+## Usage
+
+The examples below displays the use of tooltip in [Avatar](Avatar) group, [Stepper](Stepper), and [Slider](Slider).
+
+<LiveExample src='Avatar.groupstacked.tsx' truncate={false}>
+  <AllExamples.AvatarGroupOverflowTooltipExample client:load />
+</LiveExample>
+
+<LiveExample src='Stepper.tooltip.tsx'>
+  <AllExamples.StepperTooltipExample client:load />
+</LiveExample>
+
+It is not recommended to use tooltips to display critical information, or as a means to navigate an interface. They are necessary only when supplemental information is required. Functionality of the product should NOT be dependent on the user reading tooltips.
+
+<LiveExample src='Slider.tooltipnone.tsx'>
+  <AllExamples.SliderTooltipNoneExample client:load />
+</LiveExample>
+
+Tooltips are enabled by default but can be turned off.
```

Comment:
I think you meant to group the demo for "tooltip in Slider" and "tooltips can be disabled" into one example here but that would be a little confusing. Maybe we can separate it? So normal slider with tooltip belong to the first group of examples. And then another slider without tooltip to demonstrate that tooltips can be disabled. I'm not too sure, maybe we can have a second opinion on this.

Reply:
Personally I think these stepper and slider examples don't even belong on this page. Just links to those specific examples in their own pages should be enough.